src/commands/deploy/deploy.ts

```diff
@@ -92,13 +124,12 @@ async function deploy(actionContext: IActionContext, arg1: vscode.Uri | string |
         void validateGlobSettings(context, context.effectiveDeployFsPath);
     }
 
-    if (language === ProjectLanguage.CSharp && !node.site.isLinux) {
-        await updateWorkerProcessTo64BitIfRequired(context, siteConfig, node, language);
+    if (language === ProjectLanguage.CSharp && !node.site.isLinux || durableStorageType) {
```

Comment:
Does the durable storage only work with 64 bit? 

Looking below, it seems like it may be necessary for Netherite? I think it makes more sense to check for Netherite here rather than in the function definition. 

I'm assuming that you put the check in the function itself so that the warning message wouldn't come up, but is that not a valid warning even if they're using Netherite?